### ISSUE TITLE
An 'Explore' home page similar to the desktop Main landing-page

### ISSUE THREAD

**Issue by prototyperspective** [commons-app/apps-android-commons#5624]
### What is the user problem or growth opportunity you want to see solved?

The Wikimedia Commons app needs a proper Explore home page that shows interesting contents for users to have some reason to actually use (and keep using) the app and discover contents.

It could be similar to the Discover page of the Wikipedia app and maybe use parts of its code. It could incorporate tiles of the desktop home page and make it possible to add or remove which tiles are shown (like for the Wikipedia app).

Which things are shown there could be discussed separately and should be customizable anyway – I never found desktop-WMC's "Media of the day" or "Picture of the day" interesting/useful or usually reasonable for example. One could show modular Tiles like these:
* "Recently uploaded files featured in many Mediawiki pages", "Recent files added to your subscribed categories", "Highlighted content of your selected main-categories", "Highlighted categories", "Most viewed unseen media during the last 30 days", "Random quality images", "Recent audio files of music", "Highlighted audio file of music / video / … of your subscribed genre/category", "Images needing categories" and "Images needing captions" etc (if logged in), "Up-to-date [charts](https://commons.wikimedia.org/wiki/Category:Charts_showing_data_through_2022) and [maps](https://commons.wikimedia.org/wiki/Category:2022_maps_of_the_world)", "Random WMC [campaign](https://commons.wikimedia.org/wiki/Category:Images_from_Wiki_Science_Competition) images", and so on.

### How do you know that this problem exists today? Why is this important?

It's important because of these benefits:
* See the use-cases (more media that closes gaps, more contributors, more app users, retaining app users)
* Making the app useful not just for uploading media; currently it's like an app to use for three minutes and never use again (if not for uploading) that is totally unengaging and not something one would expect from the well-resourced WMF or such a large website (and the largest structured free media repo) with tens–hundreds of thousands of app-users and a few times that potential users if the app was made to be truly useful/interesting
* Improvements to this app are high-impact since most readers probably use WMC via smartphone by now and many of these use the app with many of the other mobile readers probably being open to switch to the app if they were prompted and the app was of high quality

I think this could be the most important issue in regards to WMC on mobile (with mobile rising in importance as more and more users are browsing or contributing via mobile phones).

### Who will benefit from it?

The entire open media ecosystem and online global education such as Wikipedia, Wikimedia Commons as well as WMC app users and WMC users due to better and more contributions as well as more discoverable contents. Here are some use-cases and ways of how:

* Discovering categories/subjects that miss media which they could then create/upload (I think a key goal should be to have more people upload useful media that is missing, many of these being illustrations, instead of only the 10,000th sunset photo)
* Discovering interesting contents like [Tilt–shift time-lapse videos](https://commons.wikimedia.org/wiki/Category:Tilt%E2%80%93shift_time-lapse_videos) or [Microscopic images relating to biology](https://commons.wikimedia.org/wiki/Category:Microscopic_images_relating_to_biology)
* Finding their way around WMC by enjoying to use and extensively using the app

**Comment by misaochan** [comment]
Hi @prototyperspective , thanks for your feedback! I'm curious if you have tried the "Explore" feature of the app yet, and if you have, how do you think it can be improved specifically? For instance, is it just the lack of customization that is the issue for you, or the way the UI is laid out, or the fact that it's not the "home" page?

Just to address this:
> not something one would expect from the well-resourced WMF

AFAIK the WMF is not spending any resources on this app at the moment, and hasn't been since 2022. We're maintained solely by volunteers at this time. To the best of my knowledge, all avenues for WMF funding have been closed since the restructuring of the grants system.

**Comment by prototyperspective** [comment]
Yes, I have used the Explore page of course which is why I created the issue – it merely shows a feed of random good-quality photos. I thought that page is already the default home page, at least for me it's the default page.

What can be improved specifically is what meant with "_a proper Explore home page that shows interesting contents for users to have some reason to actually use (and keep using) the app and discover contents_" in broad terms which I intended to specify with examples in the paragraph of _"Recently uploaded files featured in many Mediawiki pages"…_. 
Namely, diverse **tiles for different types of contents** just like the **Wikipedia app**. Please try out the Wikipedia app to see what I mean with tiles for different kinds of interesting contents. Likewise, the Main page of Wikimedia Commons desktop isn't a mere feed of semi-random good-quality images, it's a more useful landing page from where one can discover routes to all sorts of interesting contents such as by clicking on the category "Science". 

>AFAIK the WMF is not spending any resources on this app at the moment

It's a really horrible situation and I can't understand the WMF to any degree anymore with its millions of donated money but barely any funds for development of the piling backlog of bugs to fix and features+wishes to implement. Thanks for working on this as volunteers…even more reason for a dev-campaign-banner to get more volunteer developers involved displayed at the top of IT-related Wikipedia articles.

**Comment by nicolas-raoul** [comment]
Below are the different sections I see on my Wikipedia app home activity. Please everyone post ideas for sections that would be interesting for the Commons app. Thanks!

![Screenshot_20240315-100736.png](https://github.com/commons-app/apps-android-commons/assets/99590/e3c9805e-5ce9-4a42-8e39-a6cdd146931a)

![Screenshot_20240315-100714.png](https://github.com/commons-app/apps-android-commons/assets/99590/a72a30d3-4115-4a0e-a657-438c9cafc9f0)

![Screenshot_20240315-100655.png](https://github.com/commons-app/apps-android-commons/assets/99590/bf59f862-250e-4d5c-89a1-dd37f2aa57d5)

![Screenshot_20240315-100723.png](https://github.com/commons-app/apps-android-commons/assets/99590/1ae81390-dee2-4044-b17b-ebbaa434c098)

![Screenshot_20240315-100705.png](https://github.com/commons-app/apps-android-commons/assets/99590/a37c75f0-9736-458d-a0e4-7213b97e8f72)

![Screenshot_20240315-100749.png](https://github.com/commons-app/apps-android-commons/assets/99590/fc3b9c34-55b8-40da-bc6d-f8d972504107)

![Screenshot_20240315-100804.png](https://github.com/commons-app/apps-android-commons/assets/99590/b8737f00-654d-47c7-a690-33624b71e43c)

**Comment by prototyperspective** [comment]
Maybe it would easier/quicker to just show the Wikimedia Commons Main page until this is implemented which would be incredibly useful and is kind of expected. See the issue #5772 about that

Thanks for these screenshots, that's what I mean. However, it would be better if those images were smaller. I've edited the post to add all the ideas I had and I think the more modules/tiles people can enable there and personalize it to their liking, the more interesting the app would become and the more users there would be. Even the Wikipedia app shows a tile for "Picture of the day" but it's missing in the Commons app and just a page showing that would be good place to start implementing a feed that could then be added the the app (it doesn't have the be the landing page right away before there are more modules).

I think a Home Feed to explore is one of the three main key features – the other two being enabling uploads and  enabling browsing categories/searching. Just imagine if Twitter/X had no home feed, nobody would use it and I think the usefulness for WMC is pretty much comparable except for those who only use it to upload their smartphones photos.
